kernel/run.py

```diff
@@ -407,12 +407,17 @@ def run_qemu(args):
                           'isa-debug-exit,iobase=0xf4,iosize=0x04']
 
     if args.qemu_ivshmem:
-        # If you change the the device addr, you must change it in vibrio::rumprt::dev
-        qemu_default_args += ['-device', 'ivshmem-doorbell,vectors=3,chardev=id,addr={}'.format(IVSHMEM_DEVICE_ADDR)]
-        qemu_default_args += [
-            '-chardev',
-            'socket,path={},id=id'.format(args.qemu_shmem_path)
-        ]
+        sizes = [int(s.strip()) for s in args.qemu_ivshmem.split(",")]
+        names = [s.strip() for s in args.qemu_shmem_path.split(",")]
+        assert len(sizes) == len(names)
+
+        for i in range(len(sizes)):
+            # TODO: Only device IVSHMEM_DEVICE_ADDR will handle interrupts?
```

Comment:
Right now all devices can handle interrupts, but only one device with interrupts is needed. I don't think it's harmful to have all the devices have this functionality, so I've left it as is.